.github/workflows/release-linux.yml

```
@@ -0,0 +1,296 @@
+    # in-app verification is a follow-up that mirrors the Windows
+    # hardening work tracked alongside it.
+    runs-on: macos-15
+    needs: build
+
+    steps:
+      - uses: actions/checkout@v5
+
+      - name: Derive release metadata
+        id: meta
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:8:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)